package-lock.json

```diff
@@ -0,0 +1,1280 @@
+{
+	"name": "fast",
+	"version": "2.8.1",
```

Comment:
This doesn't have version 3.0.0.

Reply:
I'm not sure of the correct way to update this file. What's the recommended procedure?

Reply:
Re-running `npm install` with a new-enough npm should do it.  TBH I'm not totally comfortable on package-lock.json maintenance myself... especially when we commonly have so much old npm version usage still in play in various repos.